### ISSUE TITLE
vcf2maf generating core dumps, and result is missing ExAC allele counts

### ISSUE THREAD

**Issue by ckandoth** [mskcc/vcf2maf#111]
Here is a sample VCF on the luna cluster where this happens:
/ifs/res/taylorlab/jonssonp/Proj_5001_B/variants/Proj_5001_B_s_5001_NormalBlood_s_5001_PreT_mutect_calls.nochr.PASS.vcf

**Comment by ckandoth** [comment]
This appears to happen when tabix segfaults while trying to parse through the ExAC VCF. Need to implement safer wrapping of such binaries, rather than what is currently used - backticks.

**Comment by ckandoth** [comment]
I just realized that handling STDERR of the tabix command won't prevent the segfault and core dumps. The reason for this was an outdated tabix, which can be tested before using it. However, if most users install tabix as recommended in the README, and adds it to their PATHs, then this will be a niche issue. It also looks like `tabix --version` was only implemented in the rather recent `1.3.1`, while the older versions simply showed the version information in `--help`. This makes it complicated to parse out and check which version is in the user's PATH. Closing issue, but hopefully all this info is useful to anyone running into a similar problem.
